### DISCUSSION TITLE
Zipato RFID box

### DISCUSSION THREAD

**Discussion by Gahrlaag** [zwave-js/zwave-js#2327] [General]
So, I'm trying to get this zipato rfid box working in HA but there's no way to properly set codes, tags or read status on the unit (away/home) using zwave-js. Is there a way to get this thing going or..? link to the manual  -https://www.zipato.com/wp-content/uploads/2015/07/wt-rfid-Zipato-Mini-Keypad-RFiD-Z-Wave-User-Manual-v1.4.pdf 
It's listed as Wintop mini keypad RFID in the device catalog.

**Comment by AlCalzone** [comment]
I'm assuming you're using the official HA integration. If so, please open an issue in the HA repo. I'm relatively sure some of our ioBroker users also have that device and it is working.

**Reply by Gahrlaag** [reply]
Alright, I'll open in HA. Thanks

**Reply by Gahrlaag** [reply]
It's not that the device isnt' working, it's just that it lacks a lot of control functionalities such as registering a tag, code setting, alarm status etc.

**Reply by AlCalzone** [reply]
> it lacks a lot of control functionalities

You could try the zwavejs2mqtt integration. That offers a bunch more functionality in a generic way right now than the official integration which tries to hand-craft the support.
